internal/cli/up.go

```diff
@@ -62,12 +62,13 @@ func (c *upCmd) run(args []string) error {
 		return err
 	}
 
-	req := proto.CreateServiceRequest{Service: service, Watch: c.watch, LogLevel: proto.LogLevel(logLevel())}
-
-	err = dCli.CreateService(ctx, req)
-	s, ok := status.FromError(err)
-	if ok && s.Code() == codes.Unknown {
-		return errors.New(s.Message())
+	for i := range service {
+		req := proto.CreateServiceRequest{Service: service[i], Watch: c.watch, LogLevel: proto.LogLevel(logLevel())}
+		err = dCli.CreateService(ctx, req)
+		s, ok := status.FromError(err)
+		if ok && s.Code() == codes.Unknown {
+			return errors.New(s.Message())
```

Comment:
If we fail to create one service, do we want to fail to create all subsequent services?

Reply:
Is this rhetorical and a suggestion or is it a question :^)

Reply:
It's an actual question! :)

Reply:
Aborting after one failure seems like the best option right now. If replicated is starting 30 services with a single `tilt up`, it's probably better to fail early and not make them scroll up to see if things actually worked.

Reply:
Is ... that what it's doing? Sorry this sort of error handling is a little unique to me